client/webserver/site/src/css/order.scss

```diff
@@ -8,7 +8,7 @@ div.order-datum {
   font-size: 15px;
   margin: 5px 10px 5px 0;
 
-  div:first-child {
+  .order-datum-hdr {
```

Comment:
This caused over-selection for elements under those with `order-datum` class label on `/orders/{id}` page:

<img width="800" alt="image" src="https://user-images.githubusercontent.com/112318969/224531776-0a0a4af4-103f-440e-872a-3dc5c39f6fa4.png">

So, changing here, but could also be moved to a separate PR of its own (just don't think it's worth bothering with when changes aren't huge).